docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -0,0 +1,264 @@
+# Table of Contents
+
+- [Goals](#goals)
+- [Versioning Overview](#versioning-overview)
+- [Phase 1: Preparation](#phase-1-preparation)
+  - [GitHub set up](#github-set-up)
+  - [Background Research](#background-research)
+  - [Component Spec](#component-spec)
+- [Phase 2: Initial Implementation](#phase-2-initial-implementation)
+  - [Implementation](#implementation)
+  - [Documentation](#documentation)
+  - [Preview Release](#preview-release)
+- [Phase 3: Validation and Stable Release](#phase-3-validation-and-stable-release)
+  - [Tests](#tests)
+  - [Validation](#validation)
+  - [Bug Bash](#bug-bash)
+  - [Finalize documentation](#finalize-documentation)
+  - [Stable Release](#stable-release)
+
+# Goals
+
+This article outlines the steps to follow to make a component for `@fluentui/react-components`, commonly known as 'v9.' It is the successor to the `@fluentui/react` (v8) and `@fluentui/react-northstar` (v0) libraries.
+
+These components are more than just a convergence of the implementations of the v8 and v0 components. We're also taking the opportunity to:
+
+- Clean up the API and align it with the APIs used for similar components across the industry.
+- Add support for FluentUI concepts:
+  - Composition - The basic logic of the component is separated from the styling so that a user can re-compose the component to create their own modified version of the component.
+  - Slots - Supplements of the component can be replaced by the user
+  - Tokens - A theming tool common to design and engineering.
+- Make sure there is a defined upgrade path from Fabric and Northstar to the converged component.
+
+# Versioning Overview
+
+Each component is released in multiple phases. This process has changed as of July 2023, and is outlined in more detail in the [[new release process   v9 packages]] article. In summary:
```

Comment:
looks like we are missing proper link here 

`in the [[new release process   v9 packages]] article`

Reply:
Added to this PR :-)